scripts/06_makeVTK.sh

```diff
@@ -59,14 +78,15 @@ export output_name="${outputdir}/${stem}"
 #Time:          Lengthy
 
 Slicer --launch DWIToDTIEstimation \
-  --inputVolume ${inputimage} \                      #input raw DWI (must be .nrrd, no floats)
-  --outputTensor ${output_name}_DTI.nrrd \           #estimated DTI volume
-  --outputBaseline ${output_name}_SCALAR.nrrd \      #estimated baseline (non-DW) volume (i.e., the b0)
-  --enumeration WLS \                                #weighted least squares
-  --shiftNeg
+  --enumeration WLS \
```

Comment:
adding `--shiftNeg` flag back in
```suggestion
  --enumeration WLS \
  --shiftNeg \
```